docs/source/ucx.rst

```diff
@@ -1,35 +1,87 @@
 UCX Integration
 ===============
 
-Communication can be a major bottleneck in distributed systems, and that's no different in Dask and Dask-CUDA. To address that, Dask-CUDA provides integration with `UCX-Py <https://ucx-py.readthedocs.io/>`_, a Python interface for the `UCX <https://www.openucx.org/>`_ communication framework.  UCX is a low-level library that provides high-performance networking and supports several transports, including NVLink and InfiniBand for systems that have such specialized hardware, as well as TCP for those that do not.
+Communication can be a major bottleneck in distributed systems.
+Dask-CUDA addresses this by supporting integration with `UCX <https://www.openucx.org/>`_, an optimized communication framework that provides high-performance networking and supports a variety of transport methods, including:
 
+- NVLink for *something ubiquitous to NVLink systems*
+- InfiniBand for *something ubiquitous to InfiniBand systems*
+- TCP for systems that do not have specialized hardware
 
+This integration is enabled through `UCX-Py <https://ucx-py.readthedocs.io/>`_, an interface that provides Python bindings for UCX.
 
-Relevant Settings
------------------
 
-In this list below we shall see all the Dask configurations relevant for using with Dask-CUDA currently. Note that these options can also be used with mainline Dask/Distributed if you can't use Dask-CUDA for some reason. Note that many improvments for running GPU code is readily available in Dask-CUDA -- but not in mainline Dask/Distributed -- thus, we recommend using Dask-CUDA when possible.
-In the list below we describe the relevant Dask configurations for using UCX with Dask-CUDA.
+Requirements
+------------
 
-.. note::
-    These options can also be used with mainline Dask/Distributed outside of Dask-CUDA, however, we recommend using Dask-CUDA when possible. See the :doc:`Specializations for GPU Usage <specializations>` page for details on the benefits of using Dask-CUDA.
+Hardware
+^^^^^^^^
+
+*Go into more detail on what hardware is required for NVLink, InfiniBand support*
+
+Software
+^^^^^^^^
+
+*Does UCX integration require anything else other than UCX-Py?*
```

Comment:
Thanks for the link! I will also go into some detail about when you would want to use the Conda package vs building/installing from source (essentially whether or not you plan to use IB/RDMA).

Reply:
I think that information is better suited to the UCX-Py docs link I mentioned above. Let's try to avoid duplicating information which will eventually become inconsistent between the two docs.